mephisto/abstractions/blueprint.py

```diff
@@ -264,6 +264,25 @@ def get_frontend_args(self) -> Dict[str, Any]:
         """
         return self.frontend_task_config.copy()
 
+    def update_task_config_with_metadata(
+        self,
+        mephisto_data_browser,
+        task_name: str,
+        frontend_task_config: Dict[str, Any],
+    ):
+        """
+        Updates the frontend args metadata with accepted tips
+        """
+        metadata = mephisto_data_browser.get_metadata_from_task_name(task_name)
+        accepted_tips = list(
+            filter(lambda tip: tip["accepted"] == True, metadata["tips"])
+        )
+        metadata["tips"] = accepted_tips
```

Comment:
As mentioned in other comments through out, I'm not sure how I feel about leaking the tips information logic into the base blueprint class. we should be thinking in vaguer terms on the level of "metadata" or "plugins/addons" here, not specific implementations. this is a code smell

Reply:
one solution could be to move the lambda to be an argument to the `update_task_config_with_metadata` function. that way the caller specifies the metadata-specific logic, and doesn't concern the blueprint class with the knowledge. lmk if this doesn't make sense, happy to clarify

Reply:
It would still concern a blueprint class. The update_task_config_with_metadata method(allows for metadata to be exposed in the frontend through taskConfig) is called in the get_frontend_args() method of the static_react_blueprint.py, remote_procedure_blueprint.py, and the parlai_chat_blueprint.py files. 

I guess it is a good idea though, because the base blueprint file will not be using the databrowser

Reply:
This method is likely to get refactored as we'll be pulling tips from a directory instead.

Reply:
although the final UI data will be from the directory, I think we still need an approach for the helper scripts to access the metadata when the reviewer has to review the tips - so wouldn't we still need to figure this out?